util/gas.ts

```diff
@@ -0,0 +1,49 @@
+import Common from '@ethereumjs/common'
+import { BN } from 'ethereumjs-util'
+import { IOpcode } from 'types'
+
+/*
+ * Calculates dynamic gas fee
+ *
+ * @param opcode The IOpcode
+ * @param common The Common object
+ * @param inputs The Object of user inputs based on the `dynamicFee` inputs
+ *                 in the opcodes.json
+ *
+ * @returns The String representation of the gas fee.
+ *
+ * Fee calculation is based on the ethereumjs-vm functions,
+ * See: https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/vm/src/evm/opcodes/functions.ts
+ */
+
+export const calculateDynamicFee = (
+  opcode: IOpcode,
+  common: Common,
```

Comment:
We get the fork from common as well right?

Reply:
Yes, its the same common context that's used in the reference table. So, you can get the hardfork from it